src/reduxStore/selectors/accountInfoSelectors.ts

```diff
@@ -1,7 +1,6 @@
-import { RootState } from '../store';
 import { createDeepEqualSelector } from './helpers';
 
-export const accountInfoSelector = (state: RootState) => state.account;
+export const accountInfoSelector = (state: any) => state.account;
```

Comment:
`any` type should be avoided as possible. I see that before it was used `RootState`. Is this deprecated?